nengo_loihi/tests/test_testing.py

```diff
@@ -1,7 +1,32 @@
 import pytest
+import numpy as np
 
 
 def test_seed(request, seed):
     if request.config.getoption("seed_offset"):
         pytest.skip()
     assert seed == 2004919645
+
+
+def test_allclose(allclose, capsys):
```

Comment:
This test makes me realize that it doesn't make sense to compare RMSEs for tests where you're doing `assert not allclose`. I'll make a separate issue for that though.